tests/python/unittest/test_tir_schedule_compute_at.py

```diff
@@ -1353,5 +1353,154 @@ def _create_prim_func():
     verify_trace_roundtrip(sch=sch, mod=mod)
 
 
+def test_compute_at_to_early_stage():
```

Comment:
Can we make the test "unit" enough? i.e. use a minimal case for better readability.

Reply:
Thanks @Hzfengsy  I have update test case by a minimal case.